disnake/guild.py

```diff
@@ -1911,6 +1941,13 @@ async def edit(
             The new channel that is used for public updates from Discord. This is only available to
             guilds that contain ``COMMUNITY`` in :attr:`Guild.features`. Could be ``None`` for no
             public updates channel.
+        safety_alerts_channel: Optional[:class:`TextChannel`]
+            The new channel that is used for safety alerts. This is only available to
+            guilds that contain ``COMMUNITY`` in :attr:`Guild.features`. Could be ``None`` for no
```

Comment:
Just a note, looks like this can't be changed to null again once set, though that's more of an API issue and not something we should worry about.

Reply:
👏

Reply:
this is still buggy, should I keep it as optional?

Reply:
hmm. Runtime support for `None` values is fine, but the documentation should be updated until it's known whether this is an intentional limitation or not

Reply:
appears to be fixed on the API side now, so this is no longer an issue